plugins/vendor/microsoft/teams/batch/KMicrosoftTeamsDropFolderEngine.php

```diff
@@ -262,8 +262,17 @@ protected function getUpdatedFileSize (KalturaDropFolderFile $dropFolderFile)
 	protected function retrieveUserId ($creatorInfo)
 	{
 		$userInfo = $creatorInfo[MicrosoftGraphFieldNames::USER];
-		if(isset($userInfo[MicrosoftGraphFieldNames::EMAIL]))
-		{
+
+		if (isset($userInfo[MicrosoftGraphFieldNames::EMAIL])) {
```

Comment:
@vberezkin-kaltura the BE coding standard is to open a curly bracket on a new line.